inputs/abeta_sims/part_1.py

```diff
@@ -0,0 +1,365 @@
+            for i in np.arange(
+                    number_of_chains):  # iterates through the copies of chains
+                for j in np.arange(
+                        atoms_in_chain):  # itarates through the atoms in the chain
+                    print(i * (atoms_in_chain + 1) + j, i *
+                          (atoms_in_chain) + j, len(lines), i, j)
+                    # reads the old pdbfile info pertaining to the atoms of
+                    # interest
+                    a = lines[i * (atoms_in_chain) + i + j]
+                    # converts the string into a list of characters
+                    a = list(a)
+                    if (len(a) >= 21):
+                        # replaces the repetittive chain ID with unique chain
+                        # ID chosen from
+                        a[21] = ascii_uppercase[i]
+
+                    a = "".join(a)
+                    # puts the list of characters back to a string that contains line info
+                    # writes the line into the new pdbfile
+                    f.write('{}'.format(a))
+
+                    # if(int(i*(atoms_in_chain-1)+i+j)==int((atoms_in_chain-1)*i+atoms_in_chain-1+i)
+                    # and peptide_copies!=1): #checks whether the atoms is at
+                    # the end of the chain
+                    # print
+                    # int(i*atoms_in_chain+j),int((atoms_in_chain*i+atoms_in_chain-1))
+                    if(j == int(atoms_in_chain - 1)):
+                        #    print "wrote TER"
+                        # puts ter at the end of each chain
+                        f.write('TER second\n')
+                if (peptide_copies == i + 1):
+                    f.write('ENDMDL second\n')  # puts finish touches
+                    f.close()  # done
+    return output_pdbfile
+
+
+file3 = 'template.pdb'
+new_pdb_file = pdbfile_generator_w_chain_id(
+    peptide_copies, atoms_in_chain, 5, 'new_pdb.pdb', file0)
+
+directory = 'data'
+if not os.path.exists(directory):
+    os.mkdir(directory)
+template_pdb = os.path.join(directory, file3)
+
+camshift_src = '/scratch/awhite38_lab/amyloid_eds/jobs/simulation_software/simulations/amyloid_example/data/camshift.db'
+gromacs_a03_mdb_src = '/scratch/awhite38_lab/amyloid_eds/jobs/simulation_software/simulations/amyloid_example/data/a03_gromacs.mdb'
+h_src = '/scratch/damirkul/pep_simulations/sequences_amyloid/amyloid_21_30/data/Hshifts.dat'
+ha_src = '/scratch/damirkul/pep_simulations/sequences_amyloid/amyloid_21_30/data/HAshifts.dat'
+# c_src='/scratch/damirkul/pep_simulations/sequences_amyloid/amyloid/data_old/Cshifts.dat'
+# ca_src='/scratch/damirkul/pep_simulations/sequences_amyloid/amyloid/data_old/CAshifts.dat'
+# cb_src='/scratch/damirkul/pep_simulations/sequences_amyloid/amyloid/data_old/CBshifts.dat'
+# n_src='/scratch/damirkul/pep_simulations/sequences_amyloid/amyloid/data_old/Nshifts.dat'
+gromacs_a03_mdb_dst = os.path.join(directory, 'a03_gromacs.mdb')
```

Comment:
Is it possible to generalize these things instead of using absolute paths?

Reply:
I changed this to be general - taking in the location of data_filder from user and using that to get the file paths.